MoneyLoverClone/Data/Model/Event.swift

```diff
@@ -9,15 +9,15 @@
 import Foundation
 
 class Event {
-    var id: Int = 0
-    var estimateDay : Int = 0
-    var nameEvent : String = ""
-    var imgEvent : String = ""
-    var cash : Float = 0
-    var category : Category
+    var idEvent: Int = 0
```

Comment:
```suggestion
    var idEvent = 0
```
tương tự các dòng dưới